### PR TITLE
Refactored testbench code

### PR DESCRIPTION
**Pull request by silabs-hfegran**
* Moved iss-initalization to uvm_reset phase
* Moved ref_init to an interface instead of inside imperas_dv wrapper module to easily access function with uvm
* Fixed issue with ISS in regressions
* Added iss dummy package to support building with and without iss installed and remove all compile time `ifdef USE_ISS` 
* Split imperas manifest file in two, to resolve dependencies with initialization in uvm-code

Future work: change $test$plusarg("USE_ISS") to depend on uvm configuration setting.

### CODE REVIEW

File: bin/cv_regress [hunk 1]
```diff
@@ -151,6 +151,8 @@ def read_file(args, file):
             test.sub_make(args.make)
         if args.iss != None:
             test.iss = args.iss
+        else:
+            args.iss = 'YES'
```
Comment (silabs-robin) [inline bin/cv_regress:155]: Does this make USE_ISS=YES the default if no `--iss` is given?

**Review by silabs-robin — COMMENTED** [review]
I only have cosmetic comments and some questions.

File: mk/uvmt/xrun.mk [hunk 2]
```diff
@@ -73,7 +73,14 @@ XRUN_ELAB_COVFILE = -covfile $(abspath $(MAKE_PATH)/../tools/xrun/covfile.tcl)
 XRUN_RUN_COV      = -covscope uvmt_$(CV_CORE_LC)_tb \
 					-nowarn CGDEFN
 XRUN_RUN_BASE_FLAGS += -sv_lib $(DPI_DASM_LIB)
+
+# Only append the IMPERAS_DV_MODEL sv_lib flag if the file actually exists)
+ifneq (,$(wildcard $(IMPERAS_DV_MODEL)))
+ifeq ($(call IS_YES,$(USE_ISS)),YES)
 XRUN_RUN_BASE_FLAGS += -sv_lib $(IMPERAS_DV_MODEL)
+endif
+endif
```
[the remark this answers is not in the document]
Comment (silabs-hfegran) [reply mk/uvmt/xrun.mk:82]: I guess because of annoyances with editors causing tab/space issues like above

File: bin/cv_regress (hunk 1 above)
Comment (silabs-hfegran) [reply bin/cv_regress:155]: for the build-phase yes, not the run-phase (which uses the per-test setting)

File: mk/uvmt/xrun.mk (hunk 2 above)
[the remark this answers is not in the document]
Comment (silabs-hfegran) [reply mk/uvmt/xrun.mk:81]: Fixed throughout file

**Review by silabs-robin — APPROVED** [review]

**Pull request merged**